### ISSUE TITLE
The Share this Page widget X link to close the widget is not descriptive

### ISSUE THREAD

**Issue by biljanap** [wet-boew/GCWeb#360]
There is a title attribute but not all screen readers users use title attributes. There should be at least invisible text “Close Share this page”….or an image could be used with alt text.
